packages/core/src/node-utils/manifest.ts

```diff
@@ -62,6 +66,29 @@ export function patchManifest(cwd: string, data: Partial<Manifest>): boolean {
   return true;
 }
 
+export function createOrUpdateManifest(
+  cwd: string,
+  data: Partial<Manifest>
+): void {
+  const maybeManifest: FileContent<Manifest> = retrieveManifest(cwd);
```

Comment:
you should have a runtime validation on the retrieved manifest, not on the updated version of it since the data as parameter are typed controlled already so the validation should be as a first thing right at the moment you read from the file system.

Reply:
:/

Reply:
are you asking me to re-write the "file-helpers" to use io-ts?

Reply:
The file helper supports runtime validation already hence the readEntity, you have example in the code with io ts already